### PR TITLE
monorepo & create-svelte-ux

### PR DESCRIPTION
**Pull request by jycouet**

### CODE REVIEW

File: packages/create-svelte-ux/bin.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+// create the project directory if necessary
+if (dirToCreate && !fs.existsSync(projectDir)) {
+  fs.mkdirSync(projectDir);
+}
+
+const template = options_cli.template
+  ? options_cli.template
+  : await p.select({
+      message: 'Which template do you want to use?',
+      initialValue: 'react-typescript',
```
[the remark this answers is not in the document]
Comment (techniq) [reply packages/create-svelte-ux/bin.js:104]: Maybe we call that template `basic` or `barebone` or something too.

File: packages/create-svelte-ux/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "create-svelte-ux",
+  "version": "0.0.1",
+  "description": "A CLI for creating new Svelte UX projects",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jycouet/svelte-ux-monorepo",
```
Comment (techniq) [inline packages/create-svelte-ux/package.json:7]: URL :)

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -19,11 +19,19 @@ jobs:
...
         with:
-          node-version: ${{ matrix.node-version }}
-          cache: 'npm'
-      - run: npm ci
-      - run: npm run test:unit
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: 🧑‍💻 Setup env
+        uses: jycouet/jycouet/shared-config/setup@main
```
Comment (techniq) [inline .github/workflows/ci.yml:29]: Is it possible to not have this dependency?

**Review by techniq — CHANGES_REQUESTED** [review]
Quick review while I had a minute.  I'll pull it locally hopefully tonight and kick it around.  I know the Vercel build is failing...

image

I'm figuring I can just set the Root Directory to packages/svelte-ux to resolve (or similar).

image

File: .github/workflows/ci.yml (hunk 3 above)
Comment (techniq) [reply .github/workflows/ci.yml]: :) 
image

**Comment by techniq** [comment]
> Quick review while I had a minute. I'll pull it locally hopefully tonight and kick it around. I know the Vercel build is failing...
> 
> image
> 
> I'm figuring I can just set the Root Directory to packages/svelte-ux to resolve (or similar).
> 
> image

build fixed

image

File: packages/create-svelte-ux/bin.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+import { fileURLToPath } from 'node:url';
+
+// the first argument is the name of the project
+let projectDir = process.argv[2];
+let projectName = projectDir;
+
+// When it will be in the same mono repo we will be able to get the version
+// const { version } = JSON.parse(fs.readFileSync(new URL('package.json', import.meta.url), 'utf-8'));
+const version = '0.54.0';
+console.log(`${gray(`create-svelte-ux version ${version}`)}\n`);
```
[the remark this answers is not in the document]
Comment (jycouet) [reply packages/create-svelte-ux/bin.js]: Yes, it's before I did the monorepo 😅
I will do it now

**Review by techniq — APPROVED** [review]

**Pull request merged**